packages/veritone-widgets/src/widgets/MediaDetails/index.js

```diff
@@ -983,18 +990,26 @@ class MediaDetailsWidget extends React.Component {
       engineCategorySelectorItems
     } = this.getCombineAggregations();
 
-    const isPdf = /^.*\/.*pdf$/.test(
-      get(tdo, 'primaryAsset.contentType') ||
-        get(tdo, 'details.veritoneFile.mimetype')
-    );
-    const isImage = /^image\/.*/.test(
+    const tdoMimeType =
       get(tdo, 'primaryAsset.contentType') ||
-        get(tdo, 'details.veritoneFile.mimetype')
-    );
-    const isMedia = /^(audio|video)\/.*/.test(
-      get(tdo, 'primaryAsset.contentType') ||
-        get(tdo, 'details.veritoneFile.mimetype')
+      get(tdo, 'details.veritoneFile.mimetype');
+
+    const isText = includes(
+      [
```

Comment:
should probably swap these out to an array of extensions that map to contentTypes (using npm mimetype module) to be more readable